.github/workflows/cli-release.yml

```diff
@@ -0,0 +1,56 @@
+on:
+  push:
+    tags:
+      - "v1.*"
+  workflow_dispatch:
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+name: Release CLI
+jobs:
+  build:
+    name: Build ${{ matrix.os }}-${{ matrix.architecture }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        architecture: [x86_64, aarch64]
+        include:
```

Comment:
what are the additional configs here?

Reply:
It's a mechanism for adding extra config for a matching matrix variable, documented [here](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#example-adding-configurations). In this case it's defining a Rust target suffix based on the OS we're  building on.